LL/tairdelbaig.trig

```diff
@@ -183,30 +183,24 @@
 
 <#hEchthigirn>
     a irishRel:PopulationGroup ;
-    irishRel:populationGroupName ".h. Echthigirn" .    
+    irishRel:populationGroupName ".h. Echthigirn" .
 
 <#Anlón-68c6dadb>
     a foaf:Person;
     irishRel:nomName "Anlón";
     rel:childOf <#Cennétig>;
     rdfs:comment "o tat .h. Cuirc mc Anloin";
-    owl:sameAs <#Anlón>;
     irishRel:ancestorOfGroup <#hCuircmcAnloin>.
 
 <#hCuircmcAnloin>
     a irishRel:PopulationGroup ;
-    irishRel:populationGroupName ".h. Cuirc mc Anloin" .    
-
-<#hCuirc>
-    a irishRel:PopulationGroup ;
-    irishRel:populationGroupName ".h. Cuirc" .
+    irishRel:populationGroupName ".h. Cuirc mc Anloin" .
```

Comment:
Could this be better expressed with Anloin as the ancestorOf this population group?  Something like:

```
<#hCuircmcAnloin>
    a irishRel:PopulationGroup ;
    irishRel:populationGroupName ".h. Cuirc mc Anloin" .    

<#Anloin>
     a foaf:Person;
     irishRel:genName "Anloin";
     irishRel:ancestorOfGroup <#hCuircmcAnloin>.
```

Reply:
Sorted, thanks